catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/Csw.java

```diff
@@ -138,4 +142,34 @@ CswRecordCollection getRecordById(@QueryParam("") GetRecordByIdRequest request)
     @Produces({MediaType.TEXT_XML, MediaType.APPLICATION_XML})
     TransactionResponseType transaction(CswTransactionRequest request) throws CswException;
 
+
+    @DELETE
+    @Path("/subscription/{requestId}")
+    @Produces({MediaType.WILDCARD})
+    Response deleteRecordsSubscription(@PathParam("requestId") String requestId) throws CswException;
```

Comment:
Just curious. How are we planning to document this REST API? Do we expect clients to look at the JavaDoc or we'll have some other documentation to explain it? If the former, we probably want to beef up the JavaDoc comments a little.